testproject/testapp/tests.py

```diff
@@ -221,6 +222,17 @@ def testDelete(self):
         self.obj.delete()
         self.assertEqual(self.model.objects.count(), 0)
 
+    def testDeleteWithIn(self):
+        expected = self.create_multiple_models()
+        random.shuffle(expected)
+        delete_ids = expected[:random.randint(1, len(expected) - 1)]
+        count_rows = self.model.objects.all().count()
+        qs = self.model.objects.filter(id__in=delete_ids).delete()
+        after_count = self.model.objects.all().count()
+        self.assertEqual(count_rows - len(delete_ids), after_count)
+        list(self.model.objects.order_by())
```

Comment:
This line checks that queryset is fetched correctly from db. Not necessary in this test, or replace with 
```python
assertListEquals(models_left_after_delete, ...)
```